week2.html

```diff
@@ -49,7 +49,16 @@ <h3>Vector-borne diseases</h3>
 
         <ul>
           <li><a href="vbd/malaria">Malaria</a></li>
-          <li><a href="vbd/oncho">Oncho</a></li>
+          <li><a href="vbd/oncho">Oncho part 1</a></li>
+	  <li><a href="vbd/oncho_2">Oncho part 2</a></li>
```

Comment:
If the first part is `oncho2`, then the first part must be `oncho1` - please update